Tools/autotest/param_metadata/htmlemit.py

```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env python
+"""
+Emit docs in a form acceptable to the APM wordpress docs site
```

Comment:
@peterbarker Is this still true?

Even if it is, this should be ArduPilot instead of APM.